src/viam/sdk/services/motion/motion.hpp

```diff
@@ -87,6 +90,122 @@ struct motion_configuration {
 /// specific motion implementations. This class cannot be used on its own.
 class Motion : public Service {
    public:
+    /// @enum PlanState
+    /// @brief Describes the possible states a plan can be in.
+    /// @ingroup Motion
+    enum class PlanState : uint8_t {
+        in_progress = 0,
+        stopped = 1,
+        succeeded = 2,
+        failed = 3,
+    };
+
+    static PlanState from_proto(const service::motion::v1::PlanState& proto);
+    static service::motion::v1::PlanState to_proto(const PlanState& state);
+
+    /// @struct plan_status
+    /// @brief Describes the state of a given plan at a point in time.
+    /// @ingroup Motion
+    struct plan_status {
```

Comment:
Is there a consistent story for what types get PacalCase and which get snake_case? Because the enum above is PascalCase but this struct is snake.

Reply:
Just a brain fart, switching the enum to snake_case for consistency.